### PR TITLE
Add add-on support for CAS

### PR DESCRIPTION
**Pull request by pvizeli**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/1213
- Link to cli pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [x] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

**Review by mdegat01 — CHANGES_REQUESTED** [review]

File: supervisor/addons/const.py [hunk 1]
```diff
@@ -10,3 +10,4 @@ class AddonBackupMode(str, Enum):
 
 
 ATTR_BACKUP = "backup"
+ATTR_CODENOTARY = "codenotary"
```
[the remark this answers is not in the document]
Comment (mdegat01) [reply supervisor/addons/const.py:13]: Ah the idea was to intentionally tie it to that particular service. Ok then that's fine.

**Comment by ludeeus** [comment]
https://github.com/home-assistant/frontend/pull/11703

File: supervisor/addons/model.py [hunk 2]
```diff
@@ -578,6 +578,16 @@ def with_journald(self) -> bool:
         """Return True if the add-on accesses the system journal."""
         return self.data[ATTR_JOURNALD]
 
+    @property
+    def signed(self) -> bool:
+        """Return True if the image is signed."""
+        return ATTR_CODENOTARY in self.data
```
Comment (ludeeus) [inline supervisor/addons/model.py:584]:
```suggestion
        return any(service is not None for service in (self.codenotary,))
```
Then when we get more services we just add them to the tuple here

**Review by ludeeus — APPROVED** [review]

File: supervisor/addons/model.py (hunk 2 above)
Comment (pvizeli) [reply supervisor/addons/model.py:584]: yeah, it will be not the only part that have to change then :D

**Pull request merged**